\graphicspath{{chapters/images/03/}}

\chapter{Next generation sequencing}

\section{Introduction}
Next generation sequencing is the gold standard for sequencing nowadays.
A series of discoveries allowed for the development of this technology:

\begin{multicols}{2}
    \begin{itemize}
        \item $1959$: first homogeneous DNA purified.
        \item $1970$: first discovery of type $II$ restriction enzymes.
        \item $1972$: first RNA gene sequence published.
        \item $1975$: Sanger publishes his plus-minus method of sequencing, unable to distinguish homopolymers.
        \item $1977$: Maxam and Gilbert publish their method that could distinguish homopolymers.
        \item $1977$: Sanger publishes the dideoxy-sequencing method.
    \end{itemize}
\end{multicols}

    \subsection{Progresses of sequencing}
    As can be seen in the graph \ref{moore} the cost of DNA sequencing is decreasing by a greater rate than the one predicted by Moore's law.
    This allows for greater number of samples and the sequencing of a different number of genomes.

    \begin{figure}[h]
        \centering
        \includegraphics[width=0.6\textwidth]{sequencingCost}
        \caption{}
        \label{moore}
    \end{figure}

    \subsection{Methods of sequencing}
    The methods of sequencing can be grouped in three groups:

    \begin{multicols}{2}
        \begin{itemize}
          \item Chemical degradation of DNA: like the method of Maxam-Gilbert.
          \item Sequencing by synthesis (“SBS”): the most common approach and the first to be developed.
              It uses DNA polymerases in primer extension reactions.
              This technology is used by Illumina, Pacific Bioscences, Ion Torren and 454.
          \item Ligation-based: sequencing using short probes that hybridize to the template.
              This technology is used by SOLiD and Complete Genomics.
          \item Others like nanopores.
        \end{itemize}
    \end{multicols}

    \subsection{The Chain Terminators}

    \begin{figure}[H]
        \centering
        \begin{subfigure}[b]{0.49\textwidth}
            \centering
            \includegraphics[width=\textwidth]{DNA-molecule}
            \caption{Normal DNA molecule, with oxydrilic group ligated to $3'$-ends}
            \label{normalDNAaddition}
        \end{subfigure}
        \hfill
        \begin{subfigure}[b]{0.49\textwidth}
            \centering
            \includegraphics[width=\textwidth]{chain-term}
            \caption{Figure representing the difference between a normal DNA chain and one with chain terminators}
            \label{ChainTerm}
        \end{subfigure}
        \caption{Normal DNA synthesys \textit{vs} Chain terminators}
    \end{figure}

    Normally, the addition of new nucleotides to a generated molecule of DNA happens with the $3'$-end of the nucleotide chain \ref{normalDNAaddition}.
    Chain terminators are dideoxy nucleotides, ddNTPs, that cannot be further extended.
    These nucleotides don't have the oxydrilic group at their $3'$-end, so the DNA polymerase cannot further add other nucleotides to the chain \ref{ChainTerm}.

\section{Sanger method}

\begin{figure}[h]
    \centering
    \includegraphics[width=\textwidth]{Sanger}
    \caption{Sanger's method process}
    \label{Sanger}
\end{figure}

The first method ever used to sequence DNA was designed by Frederick Sanger.
The Sanger manual sequencing system consists in an \textit{in vitro} process described in figure \ref{Sanger}.
It is also named a primer extension method.
It is performed over a single-filament DNA sample, and it uses chain terminators nucleotides, one for each type of nucleobase: ddATP, ddGTP, ddTTP, ddCTP.
The reaction is done inside four different reactions tubes, each containing:

\begin{multicols}{2}
    \begin{itemize}
        \item The sample DNA to be reproduced.
        \item A DNA polymerase.
        \item The normal nucleotides.
        \item One of the four possible chain terminator marked with sulfur-35.
            In each tube, the corresponding dideoxy-nucleotide was used with a concentration 10 times lower than the other normal nucleotides.
    \end{itemize}
\end{multicols}
The polymerization reactions produces several molecules of DNA with different length: each replicative cycle is terminated after the addition of a chain terminator nucleotide.
The initial DNA sequence is reconstructed using a long PAGE gel with high concentration of urea ($6 - 7 M$) to avoid the coiling of the DNA single-filaments.
High voltages are required to achieve a highly risolutive run.
This high resolution was needed as DNA's fragments are different only for a nucleotide.
After having run it the bands were visualized through auto-radiography in order to evidentiate the phosphorescent signals.

The sequence is read starting from the shortest fragments at the end of the gel and going up along it, looking for the first presence of a band in one of the four runs.

    \subsection{Automatic sequencing}
    To automatize the Sanger methods fluorescent proteins substituted the radioactive signal.
    Several versions were developed:

    \begin{multicols}{2}
        \begin{enumerate}
          \item Fluorescent primers marked with a single fluorochrome.
          \item Four aliquotes of the same primer were used marked with four different fluorochromes, able to emit different fluorescences.
          \item Four different fluorochromes were used to mark the single ddNTPs
        \end{enumerate}
    \end{multicols}

    Thanks to the use of 4 different fluorochromes, it was possible to use a single electrophoretic lane to carry the sequencing reaction.
    Also a cyclic replicative reaction was performed with this procedure using a thermal cycler:

    \begin{multicols}{2}
        \begin{enumerate}
          \item Denaturation at $95$°C of the DNA to be sequences
          \item Annealing at $50-70$°C of the primer specific to one of the two filaments
          \item Extension at $72$°C by using a \textit{Taq}-polymerase to avoid the formation of coiled structures in the DNA molecule to be sequenced.
        \end{enumerate}
    \end{multicols}

    The resulting molecules are run through a long PAGE gel and fluorescence is triggered irradiating the DNA molecules.

    \begin{figure}[h]
        \caption{}
        \centering
        \includegraphics[width=0.8\textwidth]{automaticSang}
        \label{}
    \end{figure}

    More usefully, this sequencing method is performed by using a capillar filled with a synthetic polymer with the same function of polyacrillamide.
    The analysis produces an electropherogram, with a color depicting the probability of each base being in each position.
    The electropherogram is refined through algorithms that can boost the signal to noise ration, correct the dye effect and reduce all systematic errors.

        \begin{figure}[h]
        \caption{}
        \centering
        \includegraphics[width=0.8\textwidth]{elettroferogramma}
        \label{}
    \end{figure}

    \begin{figure}[h]
        \centering
        \includegraphics[width=0.8\textwidth]{progressSangerMachines}
        \caption{The implementation of capillary sequencing machines gave the possibility to make more runs than with the others.
                    $\tilde 1000$ fold productivity increase was allowed
                }
        \label{}
    \end{figure}

    The Sanger automatic sequencing method was used extensively for the majority of the human project.

\section{Development of Sequencing Machines}
The method used for sequencing has to be chosen based on the wanted output, like the quality required and the type of input.
Different technologies have different strengths and weaknesses:

\begin{multicols}{2}
    \begin{itemize}
        \item SOLID sequences reads long only $35$-$75$ bases and it is not used anymore.
        \item Sanger sequencing, or the capillary, can read up to $1000$ bases, but has a low throughput.
        \item MINION allowed to sequence an entire genome of \textit{E. coli}.
    \end{itemize}
\end{multicols}

A plethora of sequencing machines are available today.
None of them is able to sequence DNA directly from a sample, requiring different preparation step.
Today machines producing high-throughput output of short reads are preferred.

\begin{figure}[H]
\centering
\includegraphics[width=1\textwidth]{sequencingMachines}
\caption{It can be noticed how recent developments had the scope of increasing the output data}
\label{}
\end{figure}

The most wide-spread machines are from ILLUMINA like NovaSeq.
They use sequencing by synthesis and they amplify the signal through the use of clusters.

\begin{figure}[H]
\centering
\includegraphics[width=0.8\textwidth]{sequencingMachinesIllumina}
\caption{}
\label{}
\end{figure}

\section{Next Generation Sequencing NGS}

The NGS protocol requires $3$ steps, that are:

\begin{enumerate}
	\item \textbf{Sample preparation}: series of fragments added
	\item \textbf{Clonal amplification}: which is is needed to replicate fragments attached to the solid surfaces, since machines are not sensible to single molecules
	\item \textbf{Sequencing}: ILLUMINA sequencing is one of the techniques used to obtain sequence data nowadays
\end{enumerate}

Tools pertaining to the $3^{rd}$ generation are those that permit to read a molecule without replicating it.

\subsection{Fragments/Library preparation}
Most of the sequences sequenced are fragmented in short read sequences, since most of the machines today used aren't able to sequence reads longer than some hundreds of nucleotides.

% Fragmantation is so required to generate shorter filaments from the sequences. In the case of mRNA (cDNA libraries) the fragmentation process is not needed, as the lengths are already short enough.

The fragments obtained have to be prepeared for the sequencing process, through a process called \textbf{tagmentation}. The obtained fragments are shown in the figure. Those fragments are provided with one or two indexes, called also barcodes, two sequencing primer binding sites and regions complementary to the oligonucleotides present in the chamber (see in Clonal amplification chapter). The fragments' length has to be checked, depending on the scope of the process. Indexes are needed to run sequencing process on multiple samples, they are needed to distinguish those; when they are two, they permit to distinguisch also the 2 types of sequencing that are performed, forward and reverse.

P5 and P7 are the oligos needed to attach fragments to ILLUMINA sequencing machines.

\begin{figure}[h]
\caption{Figure representing the a good prepeared fragment, it has two indexes, two sequencing primer binding sites and regions complementary to the oligonucleotides present in the chamber}
\centering
\includegraphics[width=0.8\textwidth]{tagmentedFragments}
\label{}
\end{figure}

%what is a good fragment? %TODO

\subsection{Clonal amplification and ILLUMINA sequencing procedure}

\begin{figure}[h]
\caption{}
\centering
\includegraphics[width=0.6\textwidth]{clusterAmplification}
\label{clusters}
\end{figure}

Clonal amplification are necessary to amplify the signal from each single fragment.
ILLUMINA machines make use of clusters to sequence DNA. Clusters are a group of DNA
strands positioned closely together and generated from a single DNA filament. Generally, Each cluster represents thousands of copies of the same DNA strand in a 1–2 micron spot (figure \ref{clusters}).

\begin{figure}[h]
\caption{}
\centering
\includegraphics[width=0.8\textwidth]{rigidGeneration}
\label{rigid}
\end{figure}

On patterned flow cells, the clusters' formation location could be known or not, in the first case it is said that there is a "Rigid registration" (figure \ref{rigid}).\\


ILLUMINA sequencers normally use slides of glass, named flow cells, and the fragments to be sequenced are able to flow over the channels. The temperature inside the cells can be changed to produce ligations or separations. The surface of the cells is functionalized with a series of oligos complementary to library adapters.

Two kids of flow cells, the \textit{patterned} flow cell permits to create clusters in specific positions, inside nanowalls, contrarily to \textit{Random Flow Cell} which instead have randomly positioned clusters.
\begin{figure}[h]
\caption{}
\centering
\includegraphics[width=0.6\textwidth]{randomPatternCells}
\label{}
\end{figure}
\\

Once the fragments are made flow over the chambers, they can bind only to p5 or p7 (ILLUMINA oligos), the two oligos functionalizing the plate. Once the fragments are attached to the surface, using temperature and solvents flows you can control the sequencing process. To see the entire procedure:
(procedure on Youtube: \href{https://www.youtube.com/watch?v=womKfikWlxM}{video about ILLUMINA sequencing}). In the video, it is shown the two index sequencing process.
\\
\\


\begin{figure}[H]
    \centering

    \begin{subfigure}[b]{0.39\textwidth}
        \centering
        \includegraphics[width=\textwidth]{singleIndex}
        \caption{single index}
    \end{subfigure}
    \hfill
    \begin{subfigure}[b]{0.60\textwidth}
        \centering
        \includegraphics[width=\textwidth]{doubleIndex}
        \caption{double index}
    \end{subfigure}
    \caption{Single/double index for ILLUMINA sequencing}
    \label{singDoubInd}
\end{figure}

With 1 and 2 indexes the sequencing process actually differs, as shown in figure \ref{singDoubInd}


\begin{figure}[H]
\caption{}
\centering
\includegraphics[width=0.7\textwidth]{ILLUMINArev}
\label{ILLUMINArev}
\end{figure}

To perform the sequencing process, ILLUMINA machines utilize \textit{reversible terminators} (figure \ref{ILLUMINArev}). They permit a real time analysis of the sequencing by synthesys reaction, and because of this they are different from the Sanger method. Fluorophores are reversible, they can be cleaved to eliminate the light signal.

\begin{figure}[h]
\caption{}
\centering
\includegraphics[width=0.8\textwidth]{ChannelILLUMINA}
\label{ChannelILLUMINA}
\end{figure}

To perform their activity, ILLUMINA sequencers could be of 3 different types: 4-Channel, 2-Channel or 1-Channel, depending on the number of fluorescent molecules used. In the case of the 4-Channel technology, 4 images are taken in each cycle, and each  cluster appears in only one of four images \ref{ChannelILLUMINA}. 2-Channel technique is used by some sequencing machines, like NextSeq 550, MiniSeq, NovaSeq 6000\\

4-Colors base calling
is needed to to make
the true signal the purest, and after, The base with the highest intensity becomes the called base for that cluster. In the case no base is clearly related to a position, \textit{N} is the result.


The reading process could be done in two ways: through single reads, on a single extreme of the fragments, of paired-end, on both the extremes. The second one in particular gives structural and 2 sequence  informations.


\subsection{Pacific Bioscience (PacBio)}
the long DNA filement to be sequenced is attached to a polymerase, over the surface of a SMRT (Single Molecule Real Time) cell. This cell is really small, and at each nucleation process a light signal is emitted. The produced light is not able to get out of the walls, and its duration is extremely restricted. Sequencing, also in this case, is made by sequencing, and the main advantage consists in the possiblity of sequencing really big DNA molecules.\\
In the video \href{https://www.youtube.com/watch?v=_lD8JyAbwEo}{PacBio procedure} it is briefly shown how the process works and what are the strategies used to reduce errors.

\begin{figure}[h]
\caption{}
\centering
\includegraphics[width=1\textwidth]{PacBIO}
\label{}
\end{figure}


\subsection{Nanopore sequencing}
Intramembrane proteins are used, sequence detected through the passage of DNA nucleotides, which produce different voltage changes. During the first periods, this type of instruments gave great amount of errors, nowadays the technique is improving.
\href{https://www.youtube.com/watch?v=E9-Rm5AoZGw}{Nanopore Sequencing}
